src/gatling/scala/configurations/TrafficConfiguration.scala

```diff
@@ -7,7 +7,7 @@ import scala.math.max
 object Load {
   val standard: TrafficConfiguration = TrafficConfiguration(
     activeUsers = 40,
-    duration = 3 minutes,
+    duration = 1 minutes,
```

Comment:
why?

Reply:
To test the pipeline, with less overall duration. Now reverted back to 3 minutes.